dist/index.js

```diff
@@ -14534,49 +14525,60 @@ const core = __importStar(__webpack_require__(186));
 const github = __importStar(__webpack_require__(438));
 const yaml = __importStar(__webpack_require__(917));
 const minimatch_1 = __webpack_require__(973);
-function run() {
-    return __awaiter(this, void 0, void 0, function* () {
-        try {
-            const token = core.getInput("repo-token", { required: true });
-            const configPath = core.getInput("configuration-path", { required: true });
-            const syncLabels = !!core.getInput("sync-labels", { required: false });
-            const prNumber = getPrNumber();
-            if (!prNumber) {
-                console.log("Could not get pull request number from context, exiting");
-                return;
-            }
-            const client = new github.GitHub(token);
-            const { data: pullRequest } = yield client.pulls.get({
-                owner: github.context.repo.owner,
-                repo: github.context.repo.repo,
-                pull_number: prNumber
+async function run() {
+    try {
+        const token = core.getInput("repo-token", { required: true });
+        const configPath = core.getInput("configuration-path", { required: true });
+        const sharedConfigurations = JSON.parse(core.getInput("shared-configurations", {
+            required: true,
+        }));
+        const syncLabels = !!core.getInput("sync-labels", { required: false });
+        const prNumber = getPrNumber();
+        if (!prNumber) {
+            console.log("Could not get pull request number from context, exiting");
+            return;
+        }
+        const client = new github.GitHub(token);
+        const { data: pullRequest } = await client.pulls.get({
+            owner: github.context.repo.owner,
+            repo: github.context.repo.repo,
+            pull_number: prNumber,
+        });
+        core.debug(`fetching changed files for pr #${prNumber}`);
+        const changedFiles = await getChangedFiles(client, prNumber);
+        const localLabelGlobs = await getLabelGlobs(client, configPath);
+        const sharedConfigGlobs = await Promise.all(sharedConfigurations.map((config) => {
+            const [, repoOwner, repoName, repoPath] = config.match(/([^/]+)\/([^/]+)\/?(.*)?@/);
```

Comment:
What's the logic behind the initial comma in the array

Reply:
A `match` result always includes the FULL match as the first arg, followed by each of the capturing groups in the regex. I only want the capturing groups, so I want to skip and not use the first arg. Including it in the destructuring means the linter will complain that it's unused, so I learned a while back that this syntax is allowed and you can imply that it's there with the extra comma but not assign it to a variable name.

Reply:
Typically languages use `_` like so `[_, variable_i_want]` to give more control. @narwold with JS can you do the empty variable anywhere within the expanded array list?  Say the 2nd or 3rd argument?

Reply:
`_` in JS will actually create a variable, and the linter will complain about that too. Yes, you can do it in any arg.